docs/guides/azure-private-link-workspace-standard.md

```diff
@@ -0,0 +1,514 @@
+
+-> **Note** This guide assumes that connectivity from the on-premises user environment is already configured using ExpressRoute or a VPN gateway connection.
+
+Databricks Private Link support enables private connectivity between users and their Databricks workspaces and between clusters on the data plane and core services on the control plane within the Databricks workspace infrastructure. 
+
+You can use Terraform to deploy the underlying cloud resources and the private access settings resources automatically, using a programmatic approach. 
+
+
+This guide covers a [standard deployment](https://learn.microsoft.com/en-us/azure/databricks/administration-guide/cloud-configurations/azure/private-link-standard) to configure Azure Databricks with Private Link:
+* Two seperate VNets are used:
+  * A transit VNet 
+  * A customer Data Plane VNet
+* A private endpoint is used for back-end connectivity and deployed in the customer Data Plane VNet.
+* A private endpoint is used for front-end connectivity and deployed in the transit VNet.
+* A private endpoint is used for web authentication and deployed in the transit VNet.
+* A dedicated Databricks workspace, called Web Auth workspace, is used for web authentication traffic. This workspace is configured with the sub resource **browser_authentication** and deployed using subnets in the transit VNet.
+
+-> **Note**  
+* A seperate Web Auth workspace is not mandatory but recommended.
+* DNS mapping for SSO login callbacks to the Azure Databricks web application can either be managed by the Web Auth workspace or another workspace that is associated with the **browser_authentication** private endpoint.
+
+![Azure Databricks with Private Link - Standard deployment](https://github.com/databricks/terraform-provider-databricks/raw/master/docs/images/azure-private-link-standard.png)
+
+This guide uses the following variables:
+
+- `cidr_transit`: The CIDR for the Azure transit VNet
+- `cidr_app`: The CIDR for the Azure Data Plane VNet
+- `rg_transit`: The name of the existing resource group that will contain the Azure transit VNet and the private DNS zone for the Frontend and the Web auth private endpoint
+- `rg_app`: The name of the existing resource group that will contain the Azure Data Plane VNet and the private DNS zone for the Backend private endpoint
+- `location`: The location for Azure resources
+
+This guide is provided as-is and you can use it as the basis for your custom Terraform module.
+
+To get started with Azure Private Link integration, this guide takes you through the following high-level steps:
+
+- Initialize the required providers
+- Configure Azure objects
+  - Deploy two Azure VNets with the following subnets:
+    - Public and private subnets for Azure Databricks workspace in the Data Plane VNet
```

Comment:
if we provision two workspaces, can we make clear that we need separate subnets for each of them?